spec/level_db_spec.rb

```diff
@@ -65,6 +65,50 @@
       db.close
     end
 
+    describe '#suspend_compactions' do
+      it 'suspends compactions' do
+        db = double(:db)
+        db.stub(:suspend_compactions)
+        LevelDb::Db.new(db).suspend_compactions
+        db.should have_received(:suspend_compactions)
+      end
+    end
+
+    describe '#resume_compactions' do
+      it 'resumes compactions' do
+        db = double(:db)
+        db.stub(:resume_compactions)
+        LevelDb::Db.new(db).resume_compactions
+        db.should have_received(:resume_compactions)
+      end
+    end
+
+    describe '#compact_range' do
+      let :leveldb do
+        double(:db, compact_range: nil)
+      end
+
+      it 'compacts the given range' do
+        LevelDb::Db.new(leveldb).compact_range(from: 'one', to: 'three')
+        leveldb.should have_received(:compact_range).with('one'.to_java_bytes, 'three'.to_java_bytes)
+      end
+
+      it "is ok to omit the 'from' option" do
```

Comment:
Maybe change this to 'compacts the whole range from the given start byte when to is omitted' (and similar for the other tests). It's better to say what happens when you omit the option than saying that it's ok to omit it.